src/index.ts

```diff
@@ -33,6 +33,10 @@ import { DynatraceEnv, getDynatraceEnv } from "./getDynatraceEnv";
 
 config();
 
+// Get version from package.json
+const packageJson = require("../package.json");
```

Comment:
Using CommonJS's `require` in TypeScript is not my favorite approach. I would prefer `import` (would probably require `resolveJsonModule` or whatever it was called in `tsconfig`). Then again, any port in a storm will do.

Reply:
Good point - I will double-check that. I believe it had resolveJsonModule in tsconfig when it first proposed the change.